docs/template-contributorIntro.md

```diff
@@ -0,0 +1,19 @@
+**New Contributor Introduction**
+
+**Issue Title:** New Contributor Self-Intro [your real name]
+
+**Link to your app:**
```

Comment:
I think "Link to my app" (and also my fork, later) makes more sense, rather than "your" :p